echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/PubsubMessageHandler.java

```diff
@@ -113,8 +113,6 @@ private String makeKey(MessageDescription description, String messageId) {
 
 
   private void processEvent(MessageDescription description) {
-    log.info("Processed Pubsub event with payload {}", description.getMessagePayload());
```

Comment:
What's left in terms of logging when we process a message? This message specifically has been helpful at least for debugging, can we just s/info/debug here?

Reply:
Agree that this particular message has been helpful to me in the past.

Reply:
Sure! I'll put it on debug. I am trying in general to avoid the overload of log messages for very busy pubsub situations.

Reply:
SGTM